code/modules/farming/produce.dm

```diff
@@ -442,6 +442,24 @@
 	dropshrink = 0.9
 	rotprocess = SHELFLIFE_EXTREME
 
+/*	..................   Mandrake   ................... */
+/obj/item/reagent_containers/food/snacks/produce/mandrake
```

Comment:
i would recomend this having some kind of bad effect if harvested winout a 2 FARMING or 2 ALCHEMY . if not people is going to just spam this everywhere . instant heal is very strong and should probably be locked behind some kind of specialy

Reply:
Sounds like a good idea, I will try and figure out how to apply a skill check when harvesting the mandrakes